workshops/models.py

```diff
@@ -159,6 +159,35 @@ def self_organized_workshops_per_year_remaining(self):
 #------------------------------------------------------------
 
 
+class Sponsorship(models.Model):
+    '''Represent sponsorship from a host for an event.'''
+
+    SPONSORSHIP_LEVELS = (
+        (1000, 'Level A'),
+        (5000, 'Level B'),
+    )
+
+    organization = models.ForeignKey(
+        'Host',
+        on_delete=models.CASCADE,
+        help_text='Organization sponsoring the event'
+    )
+    event = models.ForeignKey(
+        'Event',
+        on_delete=models.CASCADE,
+    )
+    level = models.PositiveIntegerField(
+        choices=SPONSORSHIP_LEVELS,
+        default=1000,
+    )
```

Comment:
The levels defined at http://software-carpentry.org/scf/join/ list the kinds of partnership between an organization and SWC/DC. It does not list the kind of sponsorship an organization can provide for an SWC/DC workshop.
What practice does SWC/DC follow while accepting sponsorship for a particular workshop?